### PR TITLE
ram: fix zstd load on readFromZstd

### PR DESCRIPTION
**Pull request by xiaokamikami**
This will fix cases that may cause errors in other function segments because the new array is too large

### CODE REVIEW

File: src/test/csrc/common/compress.cpp [hunk 1]
```diff
@@ -200,12 +200,14 @@ long readFromZstd(void *ptr, const char *file_name, long buf_size, uint8_t load_
 
   lseek(fd, 0, SEEK_SET);
 
-  compress_file_buffer = new uint8_t[file_size];
+  long *temp_page = (long *)mallo(chunk_size);
```
Comment (klin02) [inline src/test/csrc/common/compress.cpp:203]: Pen-slip?

Comment (xiaokamikami) [reply src/test/csrc/common/compress.cpp]: really,my fault

File: src/test/csrc/common/compress.cpp [hunk 2]
```diff
@@ -200,12 +200,14 @@ long readFromZstd(void *ptr, const char *file_name, long buf_size, uint8_t load_
 
   lseek(fd, 0, SEEK_SET);
 
-  compress_file_buffer = new uint8_t[file_size];
+  long *temp_page = (long *)malloc(chunk_size);
```
Comment (xyyy1420) [inline src/test/csrc/common/compress.cpp:203]: `new long[chunk_size]` alloc `chunk_size * sizeof(long)` bytes, but `malloc(chunk_size)` just alloc `chunk_size` bytes, If I understand correctly, there should be something wrong here

**Comment by poemonsense** [comment]
For reviewers, please carefully format the squashed commit title/messages when merging this PR. Thanks

**Comment by poemonsense** [comment]
This PR is due to some exceptions during VCS simulation. However, the C++ seems to have the same functionality before and after this PR. I'm converting this to draft now. After we find why the simulation aborts on VCS, we will come back.

**Pull request closed without merging**